zookeeper/source/zk_client_common.cc

```diff
@@ -59,6 +61,23 @@ namespace zkclient {
         if (!zk->recursive_create("/work_queues/wait_for_acks", ZKWrapper::EMPTY_VECTOR, error_code)) {
             LOG(ERROR) << CLASS_NAME <<  "Failed creating /work_queues/wait_for_acks: " << error_code;
         }
+        // Ensure work_queues exist
+        if (zk->exists("/work_queues/delete", exists, error_code)){
```

Comment:
Can we use the constants instead of string literals? It would be helpful to change the other creates() to use the constants too.